src/run.rs

```diff
@@ -303,6 +304,64 @@ fn image_ids_in_use() -> io::Result<HashSet<String>> {
     Ok(image_ids)
 }
 
+// Find docker root directory.
+fn docker_root_dir() -> io::Result<String> {
+    // Query Docker for the root directory.
+    let output = Command::new("docker")
+        .args(&["info", "--format", "{{.DockerRootDir}}"])
+        .stderr(Stdio::inherit())
+        .output()?;
+
+    // Ensure the command succeeded.
+    if !output.status.success() {
+        return Err(io::Error::new(
+            io::ErrorKind::Other,
+            "Unable to determine the Docker root directory.",
+        ));
+    }
+
+    // Find the relevant line of output.
+    String::from_utf8(output.stdout)
+        .map_err(|error| io::Error::new(io::ErrorKind::Other, error))
+        .map(|s| s.trim().to_string())
+}
+
+fn prefix_match_len(s1: &str, s2: &str) -> i32 {
```

Comment:
Seems like the other functions have comments describing what they do. Can we add one for this function too? (And the one below?)

Reply:
done!